pkg/printers/internalversion/describe_test.go

```diff
@@ -1664,9 +1664,25 @@ Spec:
     From Pod Selector: id=app2,id2=app3
     From Namespace Selector: id=app2,id2=app3
     From Namespace Selector: foo in (bar1,bar2),id=app2,id2=app3
+    From IPBlock:
+        CIDR: 192.168.0.0/16
+        Except: 192.168.3.0/24, 192.168.4.0/24
     ----------
     To Port: <any> (traffic allowed to all ports)
     From: <any> (traffic not restricted by source)
+  Allowing egress traffic:
+    From Port: 80/TCP
+    From Port: 82/TCP
+    To Pod Selector: id=app2,id2=app3
+    To Namespace Selector: id=app2,id2=app3
+    To Namespace Selector: foo in (bar1,bar2),id=app2,id2=app3
+    To IPBlock:
+        CIDR: 192.168.0.0/16
+        Except: 192.168.3.0/24, 192.168.4.0/24
+    ----------
+    From Port: <any> (traffic allowed to all ports)
+    To: <any> (traffic not restricted by source)
+  Policy Types: [Ingress Egress]
```

Comment:
Shouldn't this have a comma? `[Ingress, Egress]`?

Reply:
yeah, it should. I was also trying to print it by strings.Join, but the compiler thrown an error on type cast from []PolicyTypes to []string. Should I make a manually cast or just leave it as-is?

Reply:
Hm, are there other examples of this in describe output? Main thing is to be consistent. If other slices are presented this way then that is fine.

Alternatively, this might be an option: https://play.golang.org/p/PGwTTu9HVm

Reply:
Thanks Casey for your option! There are helpers like `fooToString` in describe.go to convert such types. I modify as that way.